import/index.js

```diff
@@ -1,30 +1,54 @@
 const analyse = require('./analyze')
 const model = require('./model')
+const request = require('superagent')
 const fs = require('fs')
 
-const scanners = ['inspect', 'sslyze', 'tls', 'pageload']
+const apiKey = process.env.DATA_GOUV_API_KEY
 
+const scanners = ['inspect', 'sslyze', 'tls', 'pageload']
 const name = 'sites'
 const options = {
   dockerImage: 'betagouv/domain-scan',
   scanners,
-  file: name+ '.csv',
+  file: `${name}.csv`,
 }
 
-
-
 analyse(options, (err, stdout, stderr) => {
-  if(err) return console.error(err)
-  console.log('analyze Done')
+  if(err) {
+    console.error(err)
+    process.exit(1)
+  }
 
-  console.log('create model')
+  console.log('analyze Done')
 
   const files = scanners.map((item) => {
-    return {name: item, path: __dirname + '/../results/' + item + '.csv'}
+    return {name: item, path: `${__dirname}/../results/${item}.csv`}
   })
-  files.push({name: 'meta', path: __dirname + '/../data/' + name + '.csv'})
+
+  files.push({name: 'meta', path: `${__dirname}/../data/${name}.csv`})
 
   model(files, (data) => {
-    fs.writeFileSync(__dirname + '/../data/sites.json', JSON.stringify({data, meta: {lastUpdated: new Date().toLocaleString()}}, null, 2), 'utf8')
+    const filePath = `${__dirname}/../data/${name}.json`
+    fs.writeFileSync(filePath, JSON.stringify({data, meta: {lastUpdated: new Date().toLocaleString()}}, null, 2), 'utf8')
+
+    if (!process.env.DATA_GOUV_API_KEY) {
+      console.log('data.gouv api key not found, finished')
+      process.exit(0)
+    }
+
+    request
+      .post('http://www.data.gouv.fr/api/1/datasets/5805f1e2c751df2bb879df72/resources/234299a6-e9a4-4cf4-a220-681a94ac3e92/upload/')
```

Comment:
Add a comment to explain where this URL comes from and how it can be maintained.